cockroachcloud/managing-cmek.md

```diff
@@ -0,0 +1,143 @@
+
+1. [Create an API key](console-access-management.html#create-api-keys) for the service account to use.
+
+### Step 2. Provision your cluster
+
+1. Create a new {{ site.data.products.dedicated }} cluster. There are two ways to do this:
+	- [Using the {{ site.data.products.db }} console clusters page](https://cockroachlabs.cloud/cluster).
+	- [Using the Cloud API](cloud-api.html#create-a-new-cluster). 
+
+### Step 3. Provision IAM and KMS in your Cloud
+
+Next, you must provision the resources required resources in your Cloud, whether this is AWS or GCP:
+
+1. The cross-account IAM role (in AWS) or service account (in GCP) that {{ site.data.products.dedicated }} will use to access your key.
+
+1. The key itself.
+
+Follow the instructions depending on your cloud provider:
+
+- [Provisioning Amazon Web Services (AWS) for CMEK](cmek-ops-aws.html)
+- [Provisioning Google Cloud Platform (GCP) for CMEK](cmek-ops-gcp.html) 
+
+### Step 4. Enable CMEK for your {{ site.data.products.dedicated }} Cluster
+
+Enable CMEK for your cluster with a call to the clusters CMEK endpoint:
+
+See the [API specification](../api/cloud/v1.html#operation/CockroachCloud_EnableCMEK).
+
+{% include_cached copy-clipboard.html %}
+```shell
+curl --request POST \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/cmek \
+  --header 'Authorization: Bearer REPLACE_BEARER_TOKEN' \
+  --header 'content-type: application/json' \
+  --data '{"region_specs":[{"region":"us-central1","key_spec":{"type":"AWS_KMS","uri":"arn:aws:kms:us-west-2:111122223333:key/id-of-kms-key","auth_principal":"arn:aws:iam::account:role/role-name-with-path"}}]}'
+```
+
+## Check CMEK status
+
+An API call displays information about your cluster's use of CMEK: 
+
+See the [API specification](../api/cloud/v1.html#operation/CockroachCloud_GetCMEKClusterInfo).
+
+{% include_cached copy-clipboard.html %}
+```shell
+curl --request GET \
+  --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/cmek \
+  --header 'Authorization: Bearer REPLACE_BEARER_TOKEN'
+```
+
+## Revoking CMEK for a cluster
+
+Revoking access to the CMEK means disabling all encryption/decryption of data in your cluster, which means preventing reading and writing any data from or to your cluster. This likely implies a shutdown of your service, with significant business implications. This is a disaster mitigation tactic to be used only in a scenario involving a severe, business critical security breach.
+
+This can be done temporarily or permanently. This action is performed at the level of your Cloud Provider.
+
+### Step 1: Revoke IAM access
+
+{{site.data.alerts.callout_danger}}
+Do not delete the CMEK key.
+Deleting the CMEK key will permanently prevent decryption of your data, preventing all possible access and rendering the data inaccessible.
+{{site.data.alerts.end}}
+
+First, revoke {{ site.data.products.dedicated }}'s access to your key at the IAM level with your cloud provider. 
+
+You can do this two ways:
```

Comment:
I think you need to add a blank line here after "You can do this two ways:" because the bullets following it aren't rendering correctly.